docs/Configuration.rst

```diff
@@ -5981,7 +6036,166 @@ data is stored in the database.
 To disable the login history data storage in the database, simply set the parameter 
 ``exo.audit.login.enabled`` to ``false``.
 
+.. _Configuration.Wallet:
+
+======================
+Configuring eXo Wallet
+======================
+
+The eXo Wallet addon uses the `Ethereum Blockchain <https://www.ethereum.org/>`__ to reward users based on a crypto-currency managed inside this blockchain.
+
+Blockchain network settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The addon is configured by default to communicate with Ethereum Mainnet throught `Infura <https://infura.io/>`__ using websocket and http protocols:
+
+.. code-block:: jproperties
+
+    exo.wallet.blockchain.networkId=1
+    exo.wallet.blockchain.network.websocket=wss://mainnet.infura.io/ws/v3/a1ac85aea9ce4be88e9e87dad7c01d40
+    exo.wallet.blockchain.network.http=https://mainnet.infura.io/v3/a1ac85aea9ce4be88e9e87dad7c01d40
```

Comment:
maybe explain that the default configuration points to an eXo-owner infura account that serves as proxy to the mainnet.

Reply:
done